src/generate-gtfs-derived-assets.js

```diff
@@ -40,7 +43,29 @@ const gtfsImportConfig = {
   sqlitePath,
   ignoreDuplicates: true,  // work around fare products bug in Node-GTFS
 };
-await importGtfs(gtfsImportConfig);
+
+// Import the GTFS zip to a DB, but skip if GTFS DB already exists and is
+// newer than zip.
+const gtfsZipModificationTime = (await stat(gtfsFilePath)).mtimeMs;
+let gtfsDbModificationTime;
+try {
+  gtfsDbModificationTime = (await stat(sqlitePath)).mtimeMs;
+} catch(e) {
+  // File not found is normal. Rethrow any other errors
+  if (e?.code !== 'ENOENT' || e?.syscall !== 'stat') {
+    throw e;
+  }
+}
+if (
```

Comment:
depending on we handle generating the DB this logic may not be needed because the files will likely all be created during every container startup. so there wont ever be a DB to start.

Reply:
I added this check for development. I didn't want to wait 90 seconds for a GTFS import every time I fixed a bug in the logic later on

Reply:
In that case I would go with a simpler pattern of if the db file exists, use it, if not create it. It supports both use cases, and it is simple to understand and implement.

Reply:
That could cause confusion in development when someone updates their GTFS feed and the updates aren't reflected.